src/cli/project/export/conda.rs

```diff
@@ -0,0 +1,198 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+
+use itertools::Itertools;
+use miette::IntoDiagnostic;
+use rattler_conda_types::{MatchSpec, Platform};
+
+use crate::cli::LockFileUsageArgs;
+use crate::lock_file::UpdateLockFileOptions;
+use crate::utils::conda_environment_file::{CondaEnvDep, CondaEnvFile};
+use crate::{project, HasFeatures, Project};
+
+// enum to select version spec formatting
+#[derive(clap::ValueEnum, Clone, Debug)]
+pub enum VersionSpec {
+    Manifest,
+    Locked,
+    None,
+}
+
+/// Exports a projects dependencies as an environment.yml
+///
+/// The environment is printed to standard out
+#[derive(Debug, Parser)]
+#[clap(arg_required_else_help = false)]
+pub struct Args {
+    /// The platform to list packages for. Defaults to the current platform.
+    #[arg(long)]
+    pub platform: Option<Platform>,
+
+    /// The path to 'pixi.toml' or 'pyproject.toml'
+    #[arg(long)]
+    pub manifest_path: Option<PathBuf>,
+
+    /// The environment to list packages for. Defaults to the default environment.
+    #[arg(short, long)]
+    pub environment: Option<String>,
+
+    /// Name for environment
+    #[arg(short, long)]
+    pub name: Option<String>,
+
+    /// Dependency spec output method
+    #[arg(long, default_value = "manifest", value_enum)]
+    pub version_spec: VersionSpec,
+
+    #[clap(flatten)]
+    pub lock_file_usage: LockFileUsageArgs,
+
+    /// Don't install the environment for pypi solving, only update the lock-file if it can solve without installing.
+    #[arg(long)]
+    pub no_install: bool,
+}
+
+pub async fn execute(args: Args) -> miette::Result<()> {
+    let project = Project::load_or_else_discover(args.manifest_path.as_deref())?;
+    let environment = project.environment_from_name_or_env_var(args.environment)?;
+
+    let platform = args.platform.unwrap_or_else(|| environment.best_platform());
+
+    let name = match args.name {
+        Some(arg_name) => arg_name,
+        None => format!("{}-{}-{}", project.name(), environment.name(), platform),
+    };
+
+    let channels = environment
+        .channels()
+        .into_iter()
+        .map(|channel| channel.name().to_string())
```

Comment:
```suggestion
        .map(|channel| {
             default_channel_config().canonical_name(channel.base_url()).to_string()
        })
```

Otherwise the following channel would be exported as `conda-forge`:
```toml
channels = ["https://fast.prefix.dev/conda-forge"]
```